data/sql/updates/pending_db_world/rev_1638646536763994700.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1638646536763994700');
+
+/* Fix LK Positioning in last phase of Scarlet Enclave
+*/
+
+DELETE FROM `creature` WHERE (`id` = 29110);
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(130896, 29110, 609, 0, 0, 1, 192, 0, 0, 2310.64, -5742.08, 161.148, 3.85123, 360, 0, 0, 27890000, 0, 0, 0, 0, 0, '', 0);
```

Comment:
Seems to work better with `position_z = 161.3`

Reply:
Thanks for this.  I agree.  I'll apply it.

Reply:
It's been applied.  Give it a few more tries if you are able.

Reply:
Still not merged his feet inside the ground after a dozen more restarts.

Btw the gm state should not be related, and my mod only affects weapons models (and fixed helmet geoset on this helmet) and should therefore not interact with this issue (I guess ? Still, I tried without it, and the result is the same than with. I'll eat then try by completing the quests instead of gm on if needed, but Idon't really think so tbh).

Reply:
I mention the GM mode because it usually creates wonky results and seems like a bug (when a normal player could come across the same thing, it wouldn't be a bug)

If your tests don't make a difference with GM on, that's a plus!

Reply:
Noticed. I'll try the regular "player" way after eating (: